src/patito/polars.py

```diff
@@ -200,15 +200,15 @@ def unalias(self: LDF) -> LDF:
 
         def to_expr(va: str | AliasPath | AliasChoices) -> Optional[pl.Expr]:
             if isinstance(va, str):
-                return pl.col(va) if va in self.columns else None
+                return pl.col(va) if va in self.collect_schema().names() else None
```

Comment:
This is more efficient. It's a dictionary so this works. `.names()` materializes a list.

```suggestion
                return pl.col(va) if va in self.collect_schema() else None
```